src/components/LaunchRNAsumReport.tsx

```diff
@@ -0,0 +1,337 @@
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={() => handleCloseDialog()}
+      maxWidth={'lg'}
+      PaperProps={{ style: { overflow: 'unset', minWidth: '70vw' } }}>
+      <Container>
+        <DialogTitle>{`RNAsum Report`}</DialogTitle>
+        <DialogContent style={{ overflow: 'unset' }}>
+          <Grid
+            container
+            direction='column'
+            justifyContent='center'
+            alignItems='center'
+            spacing={3}>
+            {triggerStatus.isLoading ? (
+              <Grid item>
+                <CircularProgress />
+              </Grid>
+            ) : triggerStatus.isError ? (
+              <Grid item container direction='column' justifyContent='center' alignItems='center'>
+                <Grid item>
+                  <CancelIcon style={{ fontSize: 40, color: red[400] }} />
+                </Grid>
+                <Grid item>
+                  <Typography>Something went wrong to triggering RNAsum report</Typography>
+                </Grid>
+              </Grid>
+            ) : triggerStatus.isSuccess ? (
+              <Grid item container direction='column' justifyContent='center' alignItems='center'>
+                <Grid item>
+                  <CheckCircleIcon style={{ fontSize: 40, color: green[400] }} />
+                </Grid>
+                <Grid item>
+                  <Typography>{`Generating '${datasetSelected}' dataset RNAsum report for ${subject_id}. Check Slack for updates!`}</Typography>
+                </Grid>
+              </Grid>
+            ) : (
+              <Grid item xs={12} style={{ width: '100%' }}>
+                <Table size='small' aria-label='a dense table'>
+                  <TableBody>
+                    {/* Subject ID row */}
+                    <TableRow>
+                      <TableCell>
+                        <Typography variant='body2' display='block' gutterBottom>
+                          SUBJECT_ID
+                        </Typography>
+                      </TableCell>
+                      <TableCell align='right'>
+                        <Typography variant='body2' display='block' gutterBottom>
+                          {subject_id}
+                        </Typography>
+                      </TableCell>
+                    </TableRow>
+                    {rnasumInputData.wtsBamsIca.length < 1 ||
+                    rnasumInputData.wgsCancer.length < 1 ? (
+                      <TableRow>
+                        <TableCell>
+                          <Typography color='error'>ERROR</Typography>
+                        </TableCell>
+                        <TableCell>
+                          <Typography color='error' align='right'>
+                            {rnasumInputData.wtsBamsIca.length < 1
+                              ? `No WTS bam file found in GDS`
+                              : rnasumInputData.wgsCancer.length < 1
+                              ? `No WGS bam file found in GDS`
```

Comment:
Can you change it to say `No umccrise workflow output found for the Subject` instead, pls.